hal/src/common/thumbv6m/sercom/v2/spi.rs

```diff
@@ -1235,6 +1235,16 @@ where
         while self.sercom.spi().syncbusy.read().enable().bit_is_set() {}
         Spi { config: self }
     }
+
+    /// Enable or disable the SERCOM peripheral, and wait for the ENABLE bit to
+    /// synchronize.
+    fn enable_peripheral(&mut self, enable: bool) {
```

Comment:
I have plans to do some internal refactoring with the register-setting code in this module. I've settled on what I think is a slightly better approach. Fortunately, this function is private, so it doesn't affect my plans. I can keep the public API the same.

Reply:
Sounds good. I'll leave it as is and let you refactor in a subsequent PR.